app/views/settings/index.html.erb

```diff
@@ -181,6 +181,12 @@
 
     <h2>Miscellaneous Settings</h2>
 
+    <div class="boxline">
+      <%= f.label :show_email, "Show Email Address:",
+        :class => "required" %>
+      <%= f.check_box :show_email %>
```

Comment:
Please add a `<span class=“hint indent”>` like on Username above, with the text “Only shown to logged-in users”